db/seeds.rb

```diff
@@ -4,121 +4,40 @@
 # Parallel Tests is seeding the database, so I am skipping this in the test environment
 return if Rails.env.test?
```

Comment:
I wonder if we can instead here only run this in `Rails.env.in?(%w[review development])` to avoid duplication across the other files with this logic, just worth double checking with NPQ team if they use this somewhere else we might have missed?